### PR TITLE
fix: SDA-2571 Fixed triggering multiple BI calls

### PR DESCRIPTION
**Pull request by psjostrom**
## Description
This PR fixes multiple BI calls being triggered

[SDA-2571](https://perzoinc.atlassian.net/browse/SDA-2571)

Notes: Skalman

### CODE REVIEW

**Review by KiranNiranjan — APPROVED** [review]

File: src/app/screen-snippet-handler.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ class ScreenSnippet {
     if (isLinux) {
       this.captureUtil = '/usr/bin/gnome-screenshot';
     }
+
+    ipcMain.on('send-tracking-data-to-main', async (_event, eventData: { element: AnalyticsElements, type: ScreenSnippetActionTypes }) => {
```
Comment (VishwasShashidhar) [inline src/app/screen-snippet-handler.ts:56]: Can we change the event name here? It looks very technical. It can be something like `snippet-event`

**Review by VishwasShashidhar — CHANGES_REQUESTED** [review]
@psjostrom Please see comment.

File: src/app/analytics-handler.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ class Analytics {
      * @param eventData {IAnalyticsData}
      */
     public track(eventData: IAnalyticsData): void {
+        console.warn('sending analytics data', eventData);
```
Comment (VishwasShashidhar) [inline src/app/analytics-handler.ts:69]: this is supposed to be a logger right? Not console log?

**Review by VishwasShashidhar — APPROVED** [review]

File: src/app/analytics-handler.ts (hunk 2 above)
Comment (psjostrom) [reply src/app/analytics-handler.ts:69]: God darnit, not even a logger but was used during debugging....

**Pull request merged**